source/common/tracing/http_tracer_impl.cc

```diff
@@ -17,27 +17,25 @@
 namespace Envoy {
 namespace Tracing {
 
-static std::string buildRequestLine(const Http::HeaderMap& request_headers,
-                                    const Http::AccessLog::RequestInfo& info) {
+static std::string buildResponseCode(const Http::AccessLog::RequestInfo& info) {
```

Comment:
Can you do me a favor and just drop the following comment on top of this function:
`TODO(mattklein123) PERF: Avoid string creations/copies in this entire file.`